src/model/shared/task-interface.tsx

```diff
@@ -0,0 +1,11 @@
+import { LatLng } from 'react-native-maps';
+
+export interface Task {
+    id: string
+    ownerId: string
+    helperId?: string
+    coordinates: LatLng,
+    tags: string[]
```

Comment:
This should be an enum with the tags we currently have; Mail, Medicine, and so on.